pkg/products/amqstreams/reconciler_test.go

```diff
@@ -247,7 +248,7 @@ func TestReconciler_handleProgress(t *testing.T) {
 	for i := 0; i < 8; i++ {
```

Comment:
Just a general question. What is the 8 for? I have seen it in a three different places. Should this be changed to a const that might explain what it is and allow to be updated easier at a later date?

Reply:
My guess is that it's the number of pods amq streams creates. I'm guessing their CR don't / didn't have a status phase to tell when the product operator has finished reconciling the CR. So to check whether to product was done, there's a check for number of pods and the status of each pod :thinking: 
https://github.com/integr8ly/integreatly-operator/blob/ee804a6816587c37b55ed1295416ee2993f38ac4/pkg/products/amqstreams/reconciler.go#L217-L220

This might get fixed / changed by Cathal as part of his ticket though - https://issues.redhat.com/browse/INTLY-5140

If not, would be a good suggestion in his PR when that's opened

Reply:
Sure I will keep a look out for Cathal's pr. I was just something I had noticed and found it odd. You probable should mark this as resolved, before some asks you to change it :face_with_head_bandage: